Core.BackgroundJobs/Model/ReadModels/ScheduleItSystemUsageOverviewReadModelUpdates.cs

```diff
@@ -189,6 +190,23 @@ private int HandleProjectUpdates(CancellationToken token, int updatesExecuted, H
             return updatesExecuted;
         }
 
+        private int HandleDataProcessingRegistrationUpdates(CancellationToken token, int updatesExecuted, HashSet<int> alreadyScheduledIds)
+        {
+            foreach (var update in _updateRepository.GetMany(PendingReadModelUpdateSourceCategory.ItSystemUsage_DataProcessingRegistration, BatchSize).ToList())
+            {
+                if (token.IsCancellationRequested)
+                    break;
+
+                using var transaction = _transactionManager.Begin(IsolationLevel.ReadCommitted);
+
+                var ids = _readModelRepository.GetByDataProcessingRegistrationId(update.SourceId).Select(x => x.SourceEntityId);
```

Comment:
hvordan fanger du første binding? Bindingen dannes som en indadgående reference fra dpr til itsystem så nye bindinger bor ikke i læsemodellen. For den slags er du nok nødt til at søge i den almindelige model, for der vil tingene ikke følges ad med itsystemusage